Tests/AppTests/AnalyzerTests.swift

```diff
@@ -137,6 +143,16 @@ class AnalyzerTests: AppTestCase {
             if cmd == .gitCommitCount { return "12" }
             if cmd == .gitFirstCommitDate { return "0" }
             if cmd == .gitLastCommitDate { return "4" }
+            if cmd == .gitShortlog { return
+                """
+                10 Person 1 <person1@example.com>
+                 2 Person 2 <person2@example.com>
+                """
+            }
+
+            if cmd == .gitShortlog {
+                return "10 Person 1 <person1@example.com>"
+            }
```

Comment:
Second block can't be called, right? 👀

(you know it's good code when this is the only thing I spot 😂)

Reply:
upps! this slip in when we were analysing this. We did not want to give person 2 a chance 😂